dhash/database.py

```diff
@@ -130,10 +140,22 @@ def get(guild_id: int, channel_id: int) -> Optional[HashChannel]:
         )
         return query
 
+    @staticmethod
+    def get_limit(guild_id: int, channel_id: int) -> int:
```

Comment:
This function is not necessary. Why don't you use the `.get()` method? We're not allowing it to be anything but integer, so we don't even have to do any more checks.
```py
limit: int = HashChannel.get(guild_id, channel_id).reaction_limit
```

Reply:
I'm using it here:

```Python
        embed.add_field(
            name=_(tc, "Hint"),
            value=_(
                tc,
                "_If image is repost, give it ♻️ reaction. "
                "If it's not, click here on ❎ and when we reach {limit} reactions, "
                "this message will be deleted._",
            ).format(
                limit=HashChannel.get_limit(message.guild.id, message.channel.id),
            ),
            inline=False,
        )```
        
It's shorter.